src/Views/StatePoliciesView/StatePoliciesView.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import { getAllStatePolicies } from "../../utils/apiCalls";
+
+const StatePoliciesView = () => {
+  const [policies, setPolicies] = useState([]);
+
+  useEffect(() => {
+    getAllStatePolicies().then((data) => setPolicies(data.data.state));
```

Comment:
Just realizing that there is `.state` which isn't the JS state but the US state ahahah 🤡